libs/core/tests/unit_tests/language_models/llms/test_base.py

```diff
@@ -16,6 +18,26 @@
 )
 
 
+class InMemoryCache(BaseCache):
```

Comment:
@liugddx - I think you should put the test cases for the cache functionality in a new file - test_cache.py to be consistent with the structure of the codebase. (For example - libs/core/tests/unit_tests/language_models/chat_models/test_cache.py)
The same all the new content can delve here - libs/core/tests/unit_tests/language_models/llms/test_cache.py

Reply:
> @liugddx - I think you should put the test cases for the cache functionality in a new file - test_cache.py to be consistent with the structure of the codebase. (For example - libs/core/tests/unit_tests/language_models/chat_models/test_cache.py) The same all the new content can delve here - libs/core/tests/unit_tests/language_models/llms/test_cache.py

Thanks for your review, PTAL thanks.